File: seungbeom/23_01_11_01.java
// https://school.programmers.co.kr/learn/courses/30/lessons/120871

class Solution {
    public int solution(int n) {
        int answer = 0;

        for (int i = 0; i < n; i++) {
            answer++;
            String number = String.valueOf(answer);
            while (answer % 3 == 0 || number.contains("3")) {
                answer++;
                number = String.valueOf(answer);
            }
        }

        return answer;
    }
}

File: minseon/23_01_11_01.py
#저주의 숫자 3

def solution(n):
    x = 0
    for a in range(n):
        x += 1
        while (x % 3 == 0 or x % 10 == 3 or x//10 == 3 or x//10 == 13):
            x += 1
        print(f'a:{a} , x:{x}')
    return x
